core/src/test/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceServiceUnitTests.scala

```diff
@@ -61,10 +67,15 @@ class WorkspaceServiceUnitTests
 
   implicit val ec: scala.concurrent.ExecutionContext = scala.concurrent.ExecutionContext.global
 
-  val defaultRequestContext: RawlsRequestContext =
-    RawlsRequestContext(
-      UserInfo(RawlsUserEmail("test"), OAuth2BearerToken("Bearer 123"), 123, RawlsUserSubjectId("abc"))
+  val ctx: RawlsRequestContext = RawlsRequestContext(
```

Comment:
I got tired of having `defaultRequestContext` making practically every mock require a line break. Sorry about the resulting diff size.